tools/abel/validators/README.md

```diff
@@ -0,0 +1,192 @@
+# ABEL Spreadsheet
```

Comment:
any reason for not having this readme in the upper one?
how would the users know about it?